core/src/main/java/org/apache/carbondata/core/datamap/status/DataMapStatusManager.java

```diff
@@ -51,6 +51,16 @@ private DataMapStatusManager() {
     return storageProvider.getDataMapStatusDetails();
   }
 
+  public static boolean isDataMapEnabled(String dataMapName) throws IOException {
```

Comment:
By default all datamaps will be disabled when doing loading, but indexdatamap should be taken special case as it will be updated online along with load so  it shouldn't be disabled after load of table. But deffered build it should be disabled after load.

Reply:
ok. will be fixed